lib/prepare.js

```diff
@@ -23,6 +23,7 @@ const fs = require('fs-extra');
 const path = require('path');
 const unorm = require('unorm');
 const plist = require('plist');
+const et = require('elementtree');
```

Comment:
`cordova-ios` does not have `elementtree` package installed.

I know this works because the package exists in `cordova-common` but we should also install it to this package as a dependency.

```
npm install elementtree
```

Reply:
Ok. I add elementree in package.json.